lib/instance.js

```diff
@@ -545,16 +541,16 @@ module.exports = (function() {
                     if (err) throw err;
                   });
                 }).then(function() {
-                  delete options.skip;
+                  options.skip = undefined;
                 });
               }
             }
           });
         }
       }).then(function() {
-        if (!options.fields.length) return this;
-        if (!this.isNewRecord) return this;
-        if (!this.options.include || !this.options.include.length) return this;
+        if (!options.fields.length || !this.isNewRecord || !this.options.include || !this.options.include.length) {
```

Comment:
Bit over the recommend length of 70, if we do go for this change they should probably be folded.